src/main/java/build/buildfarm/common/io/LibC.java

```diff
@@ -25,7 +25,7 @@
 
   Pointer opendir(String s);
 
-  int closedir(Pointer dir);
+  void closedir(Pointer dir);
```

Comment:
not sure about making this void because JNI was used to match with the underlying [libc api](https://man7.org/linux/man-pages/man3/closedir.3.html).  Depending on how that binding works, this could be an issue.

Reply:
I'll roll this back just to be safe.